content/en/docs/contributing/docs/style-guide.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Documentation Style Guide
+linkTitle: Documentation Style Guide
+weight: 20
+description: >
+    Style Guide for Flux Documentation
+---
+## Documentation formatting standards
+
+This page gives writing style guidelines for the Flux documentation. These are guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+### Use upper camel case for API objects
+
+Do | Don't
+:--| :-----
+| The GitRepository resource is responsible for ... | The Git Repository resource is responsible for ... |
+
+### Use angle brackets for placeholders
+
+Use angle brackets for placeholders. Tell the reader what a placeholder represents, for example
+
+Reconcile a kustomization :
+
+```bash
+flux reconcile kustomization <name> --with-source
+```
+
+### Use bold for user interface elements
+
+Do | Don't
+:--| :-----
+Click **Fork**. | Click "Fork".
+Select **Other**. | Select "Other".
+
+### Use italics to define or introduce new terms
+
+Do | Don't
+:--| :-----
+A _Reconciliation_ is ... | A "_Reconciliation_" is ...
+
+### Use code style for filenames, directories, and paths
+
+Do | Don't
+:--| :-----
+Open the `envars.yaml` file. | Open the envars.yaml file.
+Go to the `/docs/tutorials` directory. | Go to the /docs/tutorials directory.
+Open the `/_data/concepts.yaml` file. | Open the /\_data/concepts.yaml file.
+
+### Use the international standard for punctuation inside quotes
+
+Do | Don't
+:--| :-----
+events are recorded with an associated "stage". | events are recorded with an associated "stage."
+The copy is called a "fork". | The copy is called a "fork."
+
+## Inline Code Formatting
+
+For inline code in an HTML document, use the `<code>` tag. In a Markdown
+document, use the backtick (`` ` ``).
+
+### Use code style for inline code, commands, and API objects
+
+Do | Don't
+:--| :-----
+The `flux bootstrap` command creates a `Pod`. | The "flux bootstrap" command creates a pod.
+The Helm controller manages `HelmRelease` objects… | The Helm controller manages HelmRelease objects… 
+A `Kustomization` represents … | A Kustomization represents …
+Enclose code samples with triple backticks. (\`\`\`)| Enclose code samples with any other syntax.
```

Comment:
Most of the entries are examples; from here below, they seem to be instructions. This made me pause to figure out whether the entry was in fact a self-referential example. It would be better to either stick to examples in all these tables, or stick to instructions and explicitly introduce examples as such.

Reply:
I shall reformat this to match the rest of the examples

Reply:
I have just had a read-through, and I think it's best to keep using comparison tables, even when the comparison isn't a direct opposite, for instance two lists of do's and don'ts. Keeping it this way makes the formatting consistent.

If i remove the guideline completely, I feel like we would be missing context / information

Switching between one list after the other and tables may seem a bit jarring.

Perhaps splitting up the longer tables which have both guidelines and examples into two seperate tables, may alleviate the issue?